android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java

```diff
@@ -28,33 +26,26 @@ public void animate() {
         animator.start();
     }
 
-    private Animator createAnimator(View view) {
+    private Animator createAnimator(View view, boolean show) {
```

Comment:
This is just nitpicking, but we can avoid the boolean and simply have two public methods:
`createShowAnimator(View view)`
`createHideAnimator(View view)`